cinema_service/settings.py

```diff
@@ -137,3 +137,16 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+REST_FRAMEWORK = {
+    "DEFAULT_RENDERER_CLASSES": [
+        "rest_framework.renderers.JSONRenderer",
+        "rest_framework.renderers.BrowsableAPIRenderer",
+    ],
+    "DEFAULT_PERMISSION_CLASSES": [
+        "cinema.permissons.IsAdminOrIfAuthenticatedReadOnly",
```

Comment:
There is a typo in the import path for the custom permission class. It should be `cinema.permissions.IsAdminOrIfAuthenticatedReadOnly` instead of `cinema.permissons.IsAdminOrIfAuthenticatedReadOnly`.